src/components/Expandable/Expandable.tsx

```diff
@@ -18,10 +19,16 @@ export const Expandable: React.FC<ExpandableProps> = ({
   children,
   className,
   icon = 'plus-minus',
+  virtual = false,
   ...rest
 }) => {
   const [expanded, setExpanded] = useState(__expanded__);
-  const handleClick = useCallback(() => setExpanded(expanded => !expanded), []);
+  const [showContent, setShowContent] = useState(__expanded__ && virtual ? virtual : !virtual); // to prevent cases when component is expanded and content is not loaded into the DOM
```

Comment:
This state seems unnecessary, you can just do
`<div className='Expandable__contents'>{expanded || !virtual ? children : null}</div>`